use-timescale/compression/manual-compression.md

```diff
@@ -100,11 +100,20 @@ uncompressed chunks in your data, you can roll them up into a single compressed
 chunk.
 
 To roll up your uncompressed chunks into a compressed chunk, alter the compression
-settings to set the compress chunk time interval, and run compression operations
+settings to set the compress chunk time interval and run compression operations
 to roll up the chunks while compressing.
 
+<Highlight type="note">
+The default setting of `compress_orderby` is `'time ASC'` (the ascending or ASC command is used to sort the data returned in ascending order), which causes chunks to be re-compressed
+many times during the rollup, possibly leading to a steep performance penalty. 
+Set `timescaledb.compress_orderby = 'time ASC'` to avoid this penalty.
+</Highlight>
+
+
+
 ```sql
-ALTER TABLE example SET (timescaledb.compress_chunk_time_interval = '<time_interval>');
+ALTER TABLE example SET (timescaledb.compress_chunk_time_interval = '<time_interval>',
+                         timescaledb.compress_orderby = 'time ASC');
```

Comment:
Hmmmm, weird, you are in a code block so the linter should not do this. Try indenting the first line to see if this clears the issue.

Reply:
Fixed, pending approval of workflow to re-run.